modules/programs/zsh.nix

```diff
@@ -37,6 +37,61 @@ let
     };
   };
 
+  pluginModule = types.submodule ({ config, ... }: {
+    options = {
+      src = mkOption {
+        type = types.path;
+        description = "Path to the plugin folder. Will be added to <envar>fpath</envar> and <envar>PATH</envar>.";
+      };
+
+      name = mkOption {
+        type = types.str;
+        description = "The name of the plugin. Don't forget to add <option>file</option> if the script name does not follow convention.";
```

Comment:
Looks like these two descriptions are too long, would you mind to rewrite it using `''` quotes?